packages/discovery-provider/src/tasks/entity_manager/entities/comment.py

```diff
@@ -731,6 +733,34 @@ def pin_comment(params: ManageEntityParameters):
 
     track.pinned_comment_id = comment_id
 
+    # Get the comment user_id (commenter)
+    comment_user_id = existing_comment.user_id
+    # Get the track owner_id (artist)
+    track_owner_id = track.owner_id
+
+    # Check if the track owner is verified
+    users = get_users_by_id(params.session, [track_owner_id], current_user_id=None)
+
+    if users and len(users) > 0:
```

Comment:
nit: do we need the `len(users)` check? i think `if users` confirms that it's not an empty list.